lib/osf-components/addon/components/node-metadata-form/styles.scss

```diff
@@ -0,0 +1,64 @@
+.container {
+    background-color: #fff;
+    display: inline-flex;
+    flex-direction: column;
+
+    div {
+        display: flex;
+        justify-content: space-between;
```

Comment:
These appear to be unnecessary and are causing your problems with the subjects widget.

Reply:
Ok, this is now updated for block display. I also increased the data list element since it did not reach the far right when inspected in dark mode. Thank you.